geoportal/c2cgeoportal_geoportal/views/proxy.py

```diff
@@ -102,6 +102,13 @@ def _proxy(self, url, params=None, method=None, cache=False, body=None, headers=
             headers["Forwarded"] = ",".join([headers["Forwarded"], forwarded_str])
         else:
             headers["Forwarded"] = forwarded_str
+        # Set alternative "X-Forwarded" headers
+        for forwarded_elements in headers["Forwarded"].split(","):
+            for element in forwarded_elements.split(";"):
+                key, value = element.split("=")
+                header_key = "X-Forwarded-{0!s}".format(key.lower().capitalize())
```

Comment:
```suggestion
                header_key = "X-Forwarded-{}".format(key.capitalize())
```
?

Reply:
Was useful for my local test, but probably not useful in a real use-case.
I'll remove that